app/controllers/bands_controller.rb

```diff
@@ -15,6 +15,33 @@ def create
     end
   end
 
+  def edit
+    @band = Band.find(params[:id])
+  end
+
+  def update
+    @band = Band.find(params[:id])
+
+    if @band.update(band_params)
+      flash[:notification] = "Changes have been made"
+      redirect_to root_path
+    else
+      flash.now[:error] = @band.errors.full_messages.to_sentence
+      render :edit
+    end
+  end
+
+  def destroy
+    @band = Band.find(params[:id])
```

Comment:
With more actions now would be a good time to discuss with your group about restricting user access, as in only allowing signed in users to perform actions involving creation/modification/destruction. This also ensures that only the user that created a resource as the ability to update/delete it!